### PR TITLE
Update collecting-events.adoc

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

File: docs/modules/servers/pages/distributed/configure/collecting-events.adoc [hunk 1]
```diff
@@ -60,4 +60,10 @@ A sample message looks like:
 	"sequence": "0",
 	"recurrence-id": null
 }
-....
\ No newline at end of file
+....
+
+ICALToHeader is represented by field `X-MEETING-UID` in the Header.
```
Comment (chibenwa) [inline docs/modules/servers/pages/distributed/configure/collecting-events.adoc:65]: The following pipeline positions the `X-MEETING-UID` in the Header in order for mail user agent to correlate events with this mail.

The sample look like:

...

**Comment by Arsnael** [comment]
test this please

**Review by chibenwa — COMMENTED** [review]
:+1:

**Review by Arsnael — APPROVED** [review]

**Comment by Arsnael** [comment]
Merged

**Pull request closed without merging**